google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -101,17 +101,42 @@ public BucketInfo apply(com.google.api.services.storage.model.Bucket pb) {
   private final String locationType;
   private final Logging logging;
 
+  /**
+   * Public Access Prevention enum with expected values.
+   *
+   * @see <a
+   *     href="https://cloud.google.com/storage/docs/public-access-prevention">public-access-prevention</a>
+   */
+  public enum PublicAccessPrevention {
+    ENFORCED("enforced"),
+    // Default value for Public Access Prevention
```

Comment:
A javadoc comment on the enum value will allow it to show up in the generated javadocs
```suggestion
    /**
     * Default value for Public Access Prevention
     */
```